src/SBML.jl

```diff
@@ -1,5 +1,9 @@
+"""
+$(DocStringExtensions.README)
```

Comment:
I'm not sure if displaying the whole README is a good idea (it's github-ish), will write some shorter introduction instead.